### ISSUE TITLE
Options menu too narrow on given viewport widths

### ISSUE THREAD

**Issue by pkolawa** [PJijin/Cover-Image-Generator#10]
While having viewport width < 1200px options menu becomes to narrow to operate.

Solution is to have options menu on ```position: fixed``` with possibility to expand/hide

**Comment by PJijin** [comment]
Yeah. got the issue. I think position fixed will cause issue to design area. Let me check
